wasm/python/itkwasm-elastix-emscripten/pyproject.toml

```diff
@@ -18,7 +18,6 @@ classifiers = [
   "Intended Audience :: Developers",
   "Intended Audience :: Science/Research",
   "Programming Language :: Python :: 3",
-  "Programming Language :: Python :: 3.8",
   "Programming Language :: Python :: 3.9",
   "Programming Language :: Python :: 3.10",
   "Programming Language :: Python :: 3.11",
```

Comment:
While I think it's obvious that Python 3.8 can be removed here, I wonder if it's still useful to mention all the other minor Python versions in each pyproject file. What do you think?

The root pyproject.toml file does not mention any Python version:  
https://github.com/InsightSoftwareConsortium/ITKElastix/blob/d3ca8196506fb99f7dbbff72cc08777969053c57/pyproject.toml#L36
Is that also OK?

Reply:
The minor versions are a signal to folks browsing the project on PyPI what versions of Python the project has been tested with.

Reply:
I think we benefit from explicitly communicating the versions we support :-)